client/my-sites/promote-post-i2/components/campaign-item-details/index.tsx

```diff
@@ -220,6 +220,7 @@ export default function CampaignItemDetails( props: Props ) {
 			label: __( 'Contact support' ),
 			onClick: async () => {
 				setShowErrorDialog( false );
+				// eslint-disable-next-line wpcalypso/i18n-unlocalized-url
 				window.open( 'https://wordpress.com/support/', '_blank' );
```

Comment:
by the way, you can use `import { useLocalizeUrl } from '@automattic/i18n-utils';` and then `useLocalizeUrl( 'https://wordpress.com/support/' )` to resolve this issue. Was there a different reason for using disable-line instead?

Reply:
TBH I didn't want to overcomplicate this PR (more than it was already) and dig into it at the time. 😅 Happy to start a new PR to take care of these, though!

Reply:
@noahtallen Thanks for flagging this, I've opened https://github.com/Automattic/wp-calypso/pull/79627 to follow up!

Reply:
I think it's a newer rule, so it's easy to miss!